packages/react-components/react-list-preview/stories/src/List/ListActiveElement.stories.tsx

```diff
@@ -62,13 +62,15 @@ export const ListActiveElement = () => {
   return (
     <div>
       <List
+        as="div"
```

Comment:
cannot we render it in the code <List> as "div" ? 
because what I understand now every developer who will use <List> will need to do it by themself...
And they may to forget do that...